core/base/common/FlatJaggedArray.h

```diff
@@ -57,6 +65,61 @@ namespace ttk {
       return this->offsets_[id];
     }
 
+    struct Slice {
+      const SimplexId *const ptr;
+      const SimplexId len;
+      inline SimplexId operator[](const SimplexId id) {
+#ifndef TTK_ENABLE_KAMIKAZE
+        if(id < 0 || id >= len) {
+          return -1;
+        }
+#endif
+        return ptr[id];
+      }
+      inline const SimplexId *data() const {
+        return ptr;
```

Comment:
missing `this` here and below

Reply:
Thanks! I miss a corresponding clang-tidy lint.